### PR TITLE
saxon 9.6.0.7

### PR DESCRIPTION
**Pull request by felixbuenemann**
Updated to latest version 9.6.0.7 and added a simple transform test.

### CODE REVIEW

File: Library/Formula/saxon.rb [hunk 1]
```diff
@@ -1,12 +1,36 @@
...
+  url "https://downloads.sourceforge.net/project/saxon/Saxon-HE/9.6/SaxonHE9-6-0-7J.zip"
+  sha256 "0c904b886c7512a288c38474ff5f92042b7c3c1440352f5c919aa54f3811ab66"
+  version "9.6.0.7"
 
   def install
     libexec.install Dir["*.jar", "doc", "notices"]
     bin.write_jar_script libexec/"saxon9he.jar", "saxon"
   end
+
+  def test
```
Comment (DomT4) [inline Library/Formula/saxon.rb]: `test do` rather than `def test`. The latter is deprecated.

**Comment by felixbuenemann** [comment]
@DomT4 Updated the test syntax. Is there a way to check for deprecations locally?

**Comment by DomT4** [comment]
Should be flagged by `brew audit`, I think? If not: `brew audit --strict`.

**Comment by felixbuenemann** [comment]
@DomT4 Could you retrigger the jenkins build? The mavericks failure seems erratic:
```
FATAL: Could not checkout e958193dd8d97d7340b33f4ef0bc4eb3732c574a
```

**Comment by DomT4** [comment]
Have put it back in the queue.

**Comment by felixbuenemann** [comment]
Hmm, still same error. I'll try a rebase to onto the most recent master and push again when I come home.

**Comment by DomT4** [comment]
It's good. Merged in https://github.com/Homebrew/homebrew/commit/4d6cb5dd9e2d7f4eb02f1ec44ab94e859807d424. Thank you for another contribution to Homebrew @felixbuenemann! :smiley_cat:

**Comment by felixbuenemann** [comment]
@DomT4 Thanks, seems the brew bot is a bit busy atm.

**Pull request closed without merging**